### DISCUSSION TITLE
esp32 a1s audiokit ac101-how to change microphone input gain? how to stream audio more than one destination?

### DISCUSSION THREAD

**Discussion by paralogicdr** [pschatzmann/arduino-audiokit#33] [Q&A] [answered]
Hi everyone,
I want to lower microphone input gain, how can I do this?
and I also want to stream audio not only audiokit to speaker but also audiokit to serial
is there anyone to help me

**Comment by pschatzmann** [comment, marked as answer]
>> microphone input gain
The ES8388 driver provides a method for this, but the AC101 does not. I think you will need to implement this yourself with the help of the Technical Sheet of the ES8388. If you do so, please share it..
In any case the standard approach for [Volume Control](https://github.com/pschatzmann/arduino-audio-tools/wiki/Volume-Control) should work.

>> Serial
 just write the audio to a MultiOutput which consists of a AudioKitStream and Serial
